utilities/pipelines/PSRuleValidation/Set-PSRuleGitHubOutput.ps1

```diff
@@ -41,116 +41,122 @@ function Set-PSRuleGitHubOutput {
     # Import CSV output and filter by results #
     ###########################################
 
-    $results = Import-Csv -Path $inputFilePath
+    if (-not (Test-Path $inputFilePath)) {
+        Write-Warning ('Input File [{0}] not found' -f $inputFilePath)
+        return ''
+    } else {
 
-    $passedRules = @()
-    $failedRules = @()
+        $results = Import-Csv -Path $inputFilePath
 
-    $passedRules += $results | Where-Object { $_.Outcome -EQ 'Pass' }
-    $failedRules += $results | Where-Object { $_.Outcome -EQ 'Fail' }
+        $passedRules = @()
+        $failedRules = @()
```

Comment:
```suggestion
```
Those aren't exacly needed. The below statement will also return an empty array if $results is empty

Reply:
done